lakefs_provider/hooks/lakefs_hook.py

```diff
@@ -133,3 +144,25 @@ def create_symlink_file(self, repo: str, branch: str, location: str = None)  ->
             kwargs["location"] = location
 
         return client.metadata.create_symlink_file(repository=repo, branch=branch, **kwargs)["location"]
+
+    def test_connection(self):
+        "TO do"
```

Comment:
Can we remove this line?  I think it might break pydoc, and it's also a bit confusing.

Reply:
When we test a connection of airflow the API call comes to test_connection. It  is function of airflow to test the connectivity. So, I am just overriding the functionality.  
Let me know if this is what you asked.

Reply:
Sorry for not being clearer.  I was specifically asking about l. 149, which says `"TO do"`.  If there is still something to do here, please add more text to the comment to explain what we need to do.  Otherwise, if we no longer need this line: can you just remove it?